src/server/api/index.js

```diff
@@ -2,9 +2,7 @@ require('dotenv').config();
 const express = require('express');
 const oauth = require('oauth-sign');
 const btoa = require('btoa');
-const json2xls = require('json2xls');
-const fs = require('fs');

```

Comment:
@finnzink Same with these other packages if they are no longer needed

Reply:
done. fs is used elsewhere so it needs to be kept in